### ISSUE TITLE
[menu] Mobile navigation not sufficiently apparent

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#202]
URL: https://developers.openreferraluk.org/

Amend the border colour for the navbar button `.navbar-dark .navbar-toggler` to `#ffffff` and increase the border width to 2px.

**Current**
<img width="459" alt="Screenshot 2021-07-05 at 16 58 50" src="https://user-images.githubusercontent.com/2459151/124497087-46ff8480-ddb2-11eb-9010-952b7c90cdc9.png">

**Change to**
<img width="439" alt="Screenshot 2021-07-05 at 16 58 41" src="https://user-images.githubusercontent.com/2459151/124497077-41a23a00-ddb2-11eb-9c8d-9174f6be4535.png">

**Comment by UsmanAfzal** [comment]
@matt-porism  This now only occurs on the subdomains:

- https://validator.openreferraluk.org/
- https://tools.openreferraluk.org/ApiQuery
- https://exporter.openreferraluk.org/

**Comment by matt-porism** [comment]
@UsmanAfzal I've released these apps with similar changes to those I made to the core developers site. Mobile menu border should be bolder, header image should not shrink on mobiles.
